src/control/functor.lean

```diff
@@ -69,13 +92,20 @@ instance {γ} : functor (const γ) :=
 instance {γ} : is_lawful_functor (const γ) :=
 by constructor; intros; refl
 
+instance {α β} [inhabited α] : inhabited (const α β) :=
+⟨(default _ : α)⟩
+
 end const
 
+/-- `add_const` is the functor mapping each type `α` to the constant functor `const α`. -/
```

Comment:
```suggestion
/-- `add_const α` is a synonym for the functor `const α`. 
When `α` is a monoid, `add_const α` and `const α` are applicative
functors. With `const α`, the `applicative` instance uses `α` as a
multiplicative monoid whereas `add_const α` uses the additive
monoid of `add_const α`. -/
```

Reply:
Thanks, I completely missed the point of `const` and `add_const`, and now I understand what I was looking at in `fold`.  I've taken this explanation and distributed it to `const`, `add_const`, and the module documentation.